### ISSUE TITLE
“Formula for Success- Rise Early, Work Hard, Write Code”

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#343]
## Formula for Success

[![image](https://cloud.githubusercontent.com/assets/194400/25649314/41100ff6-2fcc-11e7-8e3d-117295a3b81a.png)](https://www.goodreads.com/quotes/323582-formula-for-success--rise-early-work-hard-strike-oil)

“Formula for Success: Rise Early, Work Hard, ~~Strike Oil~~” `++ "Write Code"`

Oil was the most reliable route to riches in the 20<sup>th</sup> Century.
`code` is the fastest way  to wealth (_by far_) in the 21<sup>st</sup>.

`while` fortunes will `continue` to be made (_grown_) based on fossil fuels,
getting _started_ in drilling/extracting Oil is not _easy_ for someone without _any_ capital.
(_most of the people who are getting mega rich off Oil have **inherited** their wealth_)
By contrast learning how to `code` is _accessible_ to _everyone_.
That's why I started learning how `code` when I was a kid and why I still [***write code every day***](https://johnresig.com/blog/write-code-every-day).

## tl;dr

As a child my ***favourite book*** was the [_**Guinness Book** of **World Records**_](https://en.wikipedia.org/wiki/Guinness_World_Records) I _still_ know _most_ of the records from 1991 (_because it's the book I had when my parents took us on a **long car journey**_)

At the time [John Paul Getty Jr.](https://en.wikipedia.org/wiki/John_Paul_Getty_Jr.) was _one of_ the Richest people in the world.
see the list: http://archive.fortune.com/magazines/fortune/fortune_archive/1991/09/09/75462/index.htm

**_John_ Paul** had **_inherited_** his wealth from his father **_Jean_ Paul** whose **quote** you see ***above***.

[Jean Paul Getty](https://en.wikipedia.org/wiki/J._Paul_Getty) _was_ ***the wealthiest man*** from 1957 till his death in 1976. 

`While` Jean inherited his _initial_ wealth (_$200k_) from his father [George Getty](https://en.wikipedia.org/wiki/George_Getty) (_founder of Getty Oil_) he made ***first million*** at age 23 (_$1 million in 1916 is worth about $20 million today_).
After his father died, Jean became the president of Getty Oil and transformed it from $10M to ***$2 Billion*** in his lifetime.

At one time, his estimated wealth was roughly 1/900th of the **_entire_ U.S. economy**. 
Today that would equate to **$160 billion** -- **_four times_ Warren Buffett's net worth**.

As a kid living in a family where money was always "_tight_" the idea of being wealthy appealed.
In the early 90's I got hold of a "Fortune Magazine" (_in a news agent_) and devoured the list!
At the time Bill Gates was not yet at the top but his fortune was rising rapidly! By 1995 aged 39 Gates was the richest man. At that point I knew there was _only_ one option: I _had_ to build software.

### Further Reading

+ The ***Solitary*** Billionaire: J. Paul Getty: http://www.bbc.co.uk/programmes/p00nw1t5 (_BBC Archive Documentary - UK Only ..._)
+ ***How*** He Got Rich - the Forgotten Billionaire J. Paul Getty? http://www.gurufocus.com/news/180183/how-he-got-rich--the-forgotten-billionaire-j-paul-getty-
+ The ***Never-Boring*** Life of J. Paul Getty, World’s Richest Man http://blogs.getty.edu/iris/the-never-boring-life-of-j-paul-getty-worlds-richest-man/
+ How to Be Rich - by J Paul Getty (_himself!_) https://www.amazon.com/How-Be-Rich-Paul-Getty/dp/0515087378 (_the original `learn-...` repository!!_) 😉
